Projects/UOContent/Items/Addons/ElvenForgeAddon.cs

```diff
@@ -21,6 +21,6 @@ public ElvenForgeDeed()
         }
 
         public override BaseAddon Addon => new ElvenForgeAddon();
-        public override int LabelNumber => 1072875; // squirrel statue (east)
```

Comment:
WTF? LOL Crazy localization comment.

Reply:
This isn't accurate. I guess my push didnt push LOL